### PR TITLE
Fix fpga docs and make cli switches a list in the docs 

### PR DESCRIPTION
**Pull request by gadfort**

### CODE REVIEW

File: siliconcompiler/sphinx_ext/dynamicgen.py [hunk 1]
```diff
@@ -615,7 +615,7 @@ def document_task(self, path, toolmodule, taskmodule, taskname, toolname):
         # Annotate the target used for default values
         if chip.valid('option', 'target') and chip.get('option', 'target'):
             p = docutils.nodes.inline('')
-            target = chip.get('option', 'target')
+            target = chip.get('option', 'target').split('.')[-1]
```
Comment (troibe) [inline siliconcompiler/sphinx_ext/dynamicgen.py:618]: Why was this required?

Comment (gadfort) [reply siliconcompiler/sphinx_ext/dynamicgen.py:618]: The target gets recorded as the python module name, but the reference just needs the module name

**Comment by troibe** [comment]
I found the differences but I don't quite understand why it was failing to generate?

Before:
image
image
image
image
image
image

After:
image
image
image
image
image
image

**Review by troibe — APPROVED** [review]

**Comment by gadfort** [comment]
It fails if the fpga flow is not configured correctly.
